curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
There are additional problems when trying the local case in the background as the 'performBackgroundOperation()' method checks if the client is connected prior to executing the operation.